v1/contracts/RegistryInterface.sol

```diff
@@ -0,0 +1,27 @@
+/*
+  Registry Interface
+*/
+pragma solidity ^0.5.0;
+
+pragma experimental ABIEncoderV2;
+
+
+interface RegistryInterface {
```

Comment:
Can we move this to `common/` and import it in v1 and v0 since it isn't changing at all?

Reply:
If we go with the MultiRole for managing derivative creators, then the API does change.

Reply:
You're saying this file isn't an exact copy of `v0/contracts/RegistryInterface.sol`?

Reply:
Ahhh, that makes sense - I assumed any differences would be added methods. SGTM.